app/src/main/java/unipd/se18/ocrcamera/CameraActivity.java

```diff
@@ -1,13 +1,31 @@
 package unipd.se18.ocrcamera;
 
-import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+import android.util.Log;
+import android.widget.LinearLayout;
 
 public class CameraActivity extends AppCompatActivity {
 
+    private static final String LOG_TAG = "CameraActivity";
+
     @Override
-    protected void onCreate(Bundle savedInstanceState) {
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_camera);
+        setContentView(R.layout.camera_activity);
+        String msg = "onCreate:";
+        Log.d(LOG_TAG, msg);
+
+        //set emphasis to the corresponding activity button
+        LinearLayout linearLayout = (LinearLayout) findViewById(R.id.action_bar_linear_layout);
+        ActionBarFragment.enphasizeButton(linearLayout, R.id.action_bar_camera_button);
+    }
+
+    @Override
+    protected void onPause() {
+        String msg = "onPause:";
+        Log.d(LOG_TAG, msg);
+        super.onPause();
     }
 }
```

Comment:
I guess it's just an example of how to use enphasizeButton, but the part you wrote for this class seems fine to me, as does GalleryActivity and SettingsActivity. But there is something i don't understand: I might have totally missed it, but where is the Fragment created? Is it enough to just declare the fragment in the xml file of the activity?